json-schema/Request-Context-v2.json

```diff
@@ -0,0 +1,267 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "https://github.com/tewhatuora/schemas/json-schema/Request-Context",
+  "description": "JSON schema for request context custom header schema for apps integrating with HNZ FHIR APIs",
+  "$comment": "This schema version 2.0 extends the original Request-Context.json public schema, adding new properties.  Properties from the original version remain but some have been deprecated due to the improved support for HPI references in this version",
+  "type": "object",
+  "properties": {
+    "userIdentifier": {
+      "type": "string",
+      "description": "The alphanumeric user id as known to the system they have signed into (which is consuming the FHIR API)",
+      "example": "LJRHODE"
+    },
+    "userRole": {
+      "type": "string",
+      "enum": [
+          "PAT",
+          "PROV"
+      ]
+    },
+    "hpiPractitioner": {
```

Comment:
propose this is removed. We have secondary identifier, why not use reuse that. See comment on Secondary Identifier.